### PR TITLE
beam warmstart test

### PR DESCRIPTION
**Pull request by rajnikantsh**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1960)

### CODE REVIEW

File: src/test/scala/beam/sim/BeamWarmStartRunSpec.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      new OutputDirectoryHierarchy(outputDir, OutputDirectoryHierarchy.OverwriteFileSetting.overwriteExistingFiles)
+
+    outputDirectoryHierarchy.getIterationFilename(iterationNumber, PersonTravelTimeAnalysis.fileBaseName + ".csv")
+  }
+
+}
+
+object BeamWarmStartRunSpec {
+
+  def using[A <: AutoCloseable, B](resource: A)(f: A => B): B = {
```
Comment (REASY) [inline src/test/scala/beam/sim/BeamWarmStartRunSpec.scala]: There is already implemented `using` and other methods in [FileUtils.scala#L77](https://github.com/LBNL-UCB-STI/beam/blob/develop/src/main/scala/beam/utils/FileUtils.scala#L77), please, try to re-use them.

**Comment by JustinPihony** [comment]
Test failed:

> must run sf-light scenario for two iteration with warmstart *** FAILED *** (6 minutes, 3 seconds)
> 13:16:17   4.4321693960270965 did not equal 5.4 plus or minus 0.6 (BeamWarmStartRunSpec.scala:30)

**Comment by JustinPihony** [comment]
Can this be merged?

**Comment by JustinPihony** [comment]
Where does this stand @rajnikantsh ?

**Comment by rajnikantsh** [comment]
test!

**Comment by JustinPihony** [comment]
@rajnikantsh 

> 18:02:06 Beam WarmStart
> 18:02:06 - must run sf-light scenario for two iteration with warmstart *** FAILED *** (18 milliseconds)
> 18:02:06   java.io.FileNotFoundException: Warmstart configuration is invalid. [Population] not found at path [/home/ubuntu/workspace/beam-4ci/test/input/sf-light/warmstart]
> 18:02:06   at beam.sim.BeamWarmStart.throwErrorFileNotFound(BeamWarmStart.scala:75)
> 18:02:06   at beam.sim.BeamWarmStart.beam$sim$BeamWarmStart$$compressedLocation(BeamWarmStart.scala:59)
> 18:02:06   at beam.sim.BeamWarmStart$.updateExecutionConfig(BeamWarmStart.scala:221)
> 18:02:06   at beam.sim.BeamHelper.updateConfigWithWarmStart(BeamHelper.scala:654)
> 18:02:06   at beam.sim.BeamHelper.runBeamWithConfig(BeamHelper.scala:454)
> 18:02:06   at beam.sim.BeamHelper.runBeamWithConfig$(BeamHelper.scala:453)
> 18:02:06   at beam.sim.BeamWarmStartRunSpec.runBeamWithConfig(BeamWarmStartRunSpec.scala:12)
> 18:02:06   at beam.sim.BeamWarmStartRunSpec.$anonfun$new$2(BeamWarmStartRunSpec.scala:27)
> 18:02:06   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
> 18:02:06   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)

**Pull request merged**